### ISSUE TITLE
Wilderness areas

### ISSUE THREAD

**Issue by tmophoto** [mapbox/drone-feedback#55]
not sure how to add this data but remote controlled aircraft are not allowed in wilderness areas.  

all of the wilderness in the USA is on this map  http://storymaps.esri.com/stories/2014/wilderness-in-context/

**Comment by cstotz** [comment]
I think we should be sure to research the facts before blocking all "wilderness areas". We know that the National Parks ban them, but let's keep open areas open and maybe promote areas that are wilderness but not banned : http://www.cnn.com/2014/06/20/travel/national-park-service-drone-ban/

**Comment by aseidman** [comment]
@tmophoto what is the source for the fact that UAV are banned in the wilderness areas?

**Comment by tmophoto** [comment]
they are motorized.  you can fly over wilderness but you cannot take off or land in a wilderness area.

wilderness is non motorized.  no chainsaws, bikes, etc.

**Comment by cstotz** [comment]
It's easily find-able via Google : http://home.nps.gov/news/release.htm?id=1601

**Comment by cstotz** [comment]
And this is probably one reason why!
http://rt.com/usa/190040-yellowstone-drone-crash-ban/

**Comment by aseidman** [comment]
I believe the wilderness ban is different than the National Parks ban.

**Comment by cstotz** [comment]
I would agree with you on that. I have not seen anything regarding a "wilderness ban"...

**Comment by tmophoto** [comment]
the wilderness ban is not specific to multi rotors.  it bans anything that has a motor from taking off or landing or traveling through the wilderness area.  they have not banned them from the air (i don’t think they can do that yet) its just all the existing rules that are in place for other activities also cover any motorized (gas or electric) rc aircraft.

national parks have specific rules for remote control aircraft but this is totally different.

**Comment by evtguy** [comment]
It is most definitely illegal. I contacted several of my National Forests in my region. Among the responses was this one from the Wilderness, Wild and Scenic Rivers and Congressionally Designated Areas Program Manager:

"Hi,

A couple of things to share.  First, it is true that the FS does not regulate airspace, only the FAA does, and the 2000ft advisory is just that.  However, the Wilderness Act of 1964, (Public Law 88-557) lists some prohibitions.  Specifically, “(c) Except as specifically provided for in this Act, and subject to existing private rights, there shall be no commercial enterprise and no permanent road within any wilderness area designated by this Act and, except as necessary to meet minimum requirements for the administration of the area for the purpose of this Act (including measures required in emergencies involving the health and safety of persons within the area), there shall  be no temporary road, no use of motor vehicles, motorized equipment or motorboats, no landing of aircraft, no other form of mechanical transport, and no structure or installation within any such area” (Excerpt from the Wilderness Act, Section 4, emphasis added).

By US Forest Service (FS) policy, mechanical transport includes” “Any contrivance for moving people or material in or over land, water, or air, having moving parts, that provides a mechanical advantage to the user, and that is powered by a living or nonliving power source. This includes, but is not limited to, sailboats, hang gliders, parachutes, bicycles, game carriers, carts, and wagons. It does not include wheelchairs when used as necessary medical appliances. It also does not include skis, snowshoes, rafts, canoes, sleds, travois, or similar primitive devices without moving parts.  (Excerpt from Forest Service policy 2320.5(3), emphasis added).

Putting all this together, it has been confirmed by the Office of General Counsel that under Federal law, possessing, launching, and landing a drone/quadcopter in or from within wilderness is not legal.  Under the FAA model aircraft regulations, an individual could in theory stand outside the wilderness and fly over if they stayed within the 400 foot model aircraft ceiling and requirement to keep the drone/quadcopter in sight of the operator (per FAA regulations for model aircraft use).  We do ask people to avoid doing so, as it can have quite an impact on wildlife and the wilderness experience of others. 

That all said – are there people taking drones/UAS into wilderness to take photos and capture film?  Yes.  Is it legal? No.  We are doing our best to provide information and education and have issued just a couple of citations to date when the use was in clear violation by parties who were aware that they were engaging in illegal activity. "

Please consider adding wilderness areas to this map.

**Comment by aseidman** [comment]
@evtguy curious how did you find this data on GitHub?

**Comment by evtguy** [comment]
I sent this email to the contact email address at various national forests:

"I tried asking this question via Twitter to the national USFS account but they deferred any answer back down to the forest level.

Given the rise and popularity of drones/quadcopters with cameras or GoPro video cameras attached, have you come up with a policy regarding their use within federally designated wilderness (Mount Hood Wilderness, Salmon-Huckleberry Wilderness, etc)? If so, what rule/policy could you cite as the reason for this?

Common sense would tell me that this would be illegal but even the flight restriction languages about wilderness really only "request" that pilots stay above a certain elevation over wilderness.

Thank you!"

**Comment by BamBam849** [comment]
I would say that they fully contradicted themselves with that statement.  Seeing how a quad does not move people or material it does not fall under their statement:  "By US Forest Service (FS) policy, mechanical transport includes” “Any contrivance for moving people or material in or over land, water, or air, having moving parts, that provides a mechanical advantage to the user, and that is powered by a living or nonliving power source" (FYI the definition of transportation of material is to physically move that material from one location to another, one way).  Quads are also not a "mechanical transport" as referred to at the bottom of that paragraph since they obviously cannot transport people or material fall under nothing they referred to with the exception of the 400ft AGL limit for model aircraft which effects all of us without other documents and license.
   I jumped on this as I was talking to a US Forrest Ranger while getting the geography corrected for the area around me and he laughed his posterior off at the back-peddling nonsense quoted by evtguy (and please do not take this as any criticism of you sir) above.  The Ranger said to me that in no way does the Wilderness Act apply to model aircraft.  Only to some idiots around here flying ultralights (which you only need a drivers license to fly one???) in quite unsafe manners.
   I guess it will be up to each of us when confronted by someone to deal with it in a calm controlled manner and to use good sense.
   I'll be trying to post the Gulf Breeze, FL JSON shortly
Thank you.

**Comment by evtguy** [comment]
No offense taken but the Ranger you talked with clearly does not know or understand the regulations he's tasked with enforcing. Quads are a mechanical transport; they have an engine (mechanical) and a payload (GoPro, etc). I'll offer yet another source:

http://www.fs.fed.us/science-technology/fire/unmanned-aircraft-systems

Towards the end of the page:

"The FAA provides guidance on “Flights Over Charted U.S. Wildlife Refuges, Parks, and Forest Service Areas”. Per this guidance, federal laws prohibit certain types of flight activity and/or provide altitude restrictions over “designated Forest Service Areas.”  UAS are considered to be  "mechanized” equipment and cannot take off and land in designated Wilderness Areas on National Forest System lands."

Look- I would love to use a drone in my photography so I don't necessarily have anything against them. The areas I would love to use them most is in Wilderness but- it's prohibited. The rules are the rules, no matter how much we may not like them. And using them in Wilderness is a rule that is NOT going to change.
